scripts/create_output_supplemental_data.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import re
+
+def add_release_version(doxyfile_path, data_obj):
```

Comment:
Might be a bit "cleaner" if this was named `get_release_version` and just did `return version`, and then in the "main" you could do `data_obj = { 'pico_sdk_release': get_release_version(input_file) }` ?

Reply:
Good idea!